packages/devtools_server/lib/src/server.dart

```diff
@@ -106,6 +136,9 @@ Future<HttpServer> serveDevToolsWithArgs(List<String> arguments,
     headlessMode: headlessMode,
     numPortsToTry: numPortsToTry,
     handler: handler,
+    observatoryAuth: authenticationUrl,
```

Comment:
call this serviceProtocolUri
instead of observatoryAuth here and elsewhere to keep the naming clear. we want to move people away from thinking of this as an observatory uri and towards thinking of it as the Dart VM service uri. I'm fine with a short vm related name as well. Could be just vmUri or something.

Reply:
Much better names. Done.